code/modules/atmospherics/FEA_gas_mixture.dm

```diff
@@ -292,7 +292,9 @@ What are the archived variables for?
 
 	#define _REMOVE_GAS(GAS, ...) \
 		removed.GAS = QUANTIZE((GAS/sum)*amount); \
-		GAS -= removed.GAS/group_multiplier;
+		GAS -= removed.GAS/group_multiplier; \
+		removed.GAS -= min(GAS, 0);  \
```

Comment:
hol up, why does this work? can GAS be negative?

Reply:
Quantization can rounding up to above the actual amount, that was the cause of the #432 with negative readings I believe.

On re-re-visit I think the first solution is correct.  Attempting to use your typo, because it seemed like a fascinating solution, doesn't actually improve anything is and is hard to read.